### PR TITLE
AI Launch 2023 - Announcement banners

### PR DESCRIPTION
**Pull request by kelbyhawn**
Add side-by-side announcement banners on Learn and Teach pages to support the launch of TeachAI:

**Learn page:** 
- Student logged out - https://studio.code.org/courses
- Student logged in - https://studio.code.org/home
- Includes translated strings

**Teach page:** 
- Teacher logged out - https://studio.code.org/courses?view=teacher
- Teacher logged in - https://studio.code.org/home
  - Added PL Superhero skinny banner under announcement banner
- English only since it's using `announcements.json`

**Related PRs:**
- https://github.com/code-dot-org/code-dot-org/pull/51462
- https://github.com/code-dot-org/code-dot-org/pull/51415
- https://github.com/code-dot-org/code-dot-org/pull/51448
- https://github.com/code-dot-org/code-dot-org/pull/51927

**Jira ticket:** https://codedotorg.atlassian.net/browse/ACQ-540

----

### Logged out
Student - Logged Out

### Logged in
Student - Logged In

### Logged out
Teach - Logged Out

### Logged in
Teach - Logged In

### CODE REVIEW

File: apps/src/templates/studioHomepages/TeacherHomepage.jsx [hunk 1]
```diff
@@ -201,6 +202,7 @@ export const UnconnectedTeacherHomepage = ({
             solidBorder={true}
           />
         )}
+        <ProfessionalLearningSkinnyBanner />
```
Comment (kelbyhawn) [inline apps/src/templates/studioHomepages/TeacherHomepage.jsx:205]: Adds the PL skinny banner to the logged in Teacher homepage.

File: pegasus/sites.v3/code.org/announcements.json [hunk 2]
```diff
@@ -2,7 +2,8 @@
   "pages": {
     "/projects": "coldplay",
     "/certificates": "hoc-more",
-    "/home": "superhero"
+    "/home": "ai-launch-2023",
+    "/courses": "ai-launch-2023"
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/announcements.json:6]: Adds the banner to https://studio.code.org/courses?view=teacher

File: pegasus/sites.v3/code.org/announcements.json [hunk 3]
```diff
@@ -2,7 +2,8 @@
   "pages": {
     "/projects": "coldplay",
     "/certificates": "hoc-more",
-    "/home": "superhero"
+    "/home": "ai-launch-2023",
```
Comment (kelbyhawn) [inline pegasus/sites.v3/code.org/announcements.json:5]: Adds the banner to https://studio.code.org/home

**Review by megcrenshaw — APPROVED** [review]
Looks great! Thanks for the screenshots

**Pull request merged**